adding cumsum function to torch frontend

vishwaksenan opened the pull request:
Hi. This PR links to the issue #3156. This PR aims to add cumsum function pytorch's frontend api. I have also updated .gitignore to add VSCode's local files ignore.

kimbring2 commented:
@vishwaksenan Hello, you need to add the 'dtype', 'out' argument to the test_frontend_function. Additionally, the cumsum function of Torch does not work with 'float16'. You need to exclude it from the available_dtypes.

vishwaksenan commented:
Hi @kimbring2. Sorry that I didn't notice that torch's cumsum doesn't support 'float16' and I have added that in the unsupported_dtype. I have also added the `dtype` and `out` parameter in the `test_frontend_function`. Although I am quite confused on how to add the `out` argument as `test_frontend_function` already has `with_out` parameter.

When I change the `with_out` parameter to True, it is not capturing the out keyword argument of the function. Can you help me  clear this doubt?

kimbring2 commented:
@vishwaksenan Hello, your code can pass the test (: Thank you for your contribution.
